docs/services/aws-services-guide.md

```diff
@@ -96,7 +103,7 @@ EKS_SECURITY_GROUP=$(aws cloudformation describe-stacks --output json --stack-na
 
 #### Create an IAM Role for the EKS cluster
 
-For EKS cluster to be able to access different resources, it needs to be given the required permissions through an **IAM Role**. In this section we create a role and assign the required policies. Later we will make EKS to assume this role.
+For the EKS cluster to be able to access different resources, it needs to be given the required permissions through an **IAM Role**. In this section we create a role and assign the required policies. Later we will make EKS to assume this role.
```

Comment:
Nitpick: `make EKS to assume this role` should be more like `make EKS assume this role`, or `assume this role from EKS`